M1/L2/index.html

```diff
@@ -60,15 +60,15 @@ <h3>The error message is your friend</h3>
 
     <p>This process is not necessarily intuitive and may take some time to learn. But after a while, you will develop a sense of the best words to use to search to get the best results. You'll also start to recognize websites that seem well-structured and informative.</p>
 
-    <p>Most despairingly, typically solving one problem will lead to another problem surfacing. Take a deep breath and repeat the process on <i>that</i> problem. But be sure to remember the issues as they arise. The knowledge you gain from solving one problem on a project stands to benefit any future problems.</p>
```

Comment:
I know it's a bit wordy, but I think it's important to clarify it's not all the time. Would you mind removing this change?

Reply:
added back the original text